apps/zbugs/src/pages/issue/issue-composer.tsx

```diff
@@ -49,18 +51,25 @@ export function IssueComposer({isOpen, onDismiss}: Props) {
     });
   }, [description]);
 
-  const handleSubmit = () => {
+  const handleSubmit = async () => {
     const id = nanoid();
 
-    z.mutate.issue.create({
+    const result = z.mutate.issue.create({
       id,
       title,
       description: description ?? '',
       created: Date.now(),
       modified: Date.now(),
     });
+
     reset();
     onDismiss(id);
+
+    const raceResult = await promiseRace([sleep(5000), result.server]);
+    if (raceResult === 0) {
+      // TODO show toast
```

Comment:
Do we want this to be a toast?

Reply:
This makes sense - agreed it would be much better to have this in `onError`. So I can add this toast later and only have this PR include online indicator, once the onError handler lands.